Assets/Plugins/Source/Editor/ConfigEditors/ConfigEditor.cs

```diff
@@ -189,5 +283,9 @@ public async Task RenderAsync()
             RenderContents();
         }
 
+        public void Dispose()
```

Comment:
Should the Action assigned to OnExpanded be unset here, so that GC can get it? Maybe Actions don't work that way.

Reply:
No, not in this case. The action is not considered a "resource" that needs freeing, it is a value that is assigned to the instance of the `ConfigEditor`, and can only be called with an instance reference.